lib/transport/transport-tls.c

```diff
@@ -239,6 +241,15 @@ log_transport_tls_write_method(LogTransport *s, const gpointer buf, gsize buflen
   return -1;
 }
 
+TLSSession *
+log_tansport_tls_get_session(LogTransport *s)
+{
+  g_assert(s->name == TLS_TRANSPORT_NAME);
```

Comment:
nitpick: I don't think C compilers will always ensure that the pointer value of a literal string expression is always the same. I would have used the address of one of the private methods instead.

Reply:
We wanted something that will less likely change if we happen to inherit from the TLS clss.

Using the address of `const gchar *a = "static"` and `const gchar a[] = "static"` should be safe and consistent in C99.